scripts/run_auto-sklearn_for_metadata_generation.py

```diff
@@ -139,29 +139,21 @@
             stats.submitted_ta_runs += 1
             stats.finished_ta_runs += 1
             memory_lim = memory_limit_factor * automl_arguments['memory_limit']
-            ta = ExecuteTaFuncWithQueue(backend=automl.automl_._backend,
-                                        port=None,
-                                        autosklearn_seed=seed,
-                                        resampling_strategy='test',
-                                        memory_limit=memory_lim,
-                                        disable_file_output=True,
-                                        logger=logger,
-                                        stats=stats,
-                                        scoring_functions=scoring_functions,
-                                        include=include,
-                                        metric=automl_arguments['metric'],
-                                        cost_for_crash=get_cost_of_crash(automl_arguments['metric']),
-                                        abort_on_first_run_crash=False,
-                                        pynisher_context='spawn')
-            run_info, run_value = ta.run_wrapper(
-                RunInfo(
-                    config=config,
-                    instance=None,
-                    instance_specific=None,
-                    seed=1,
-                    cutoff=per_run_time_limit*3,
-                    capped=False,
-                )
+
+            pipeline, run_info, run_value = automl.fit_pipeline(
+                X=X_train, y=y_train, dataset_name=dataset_name,
+                feat_type=cat, X_test=X_test, y_test=y_test,
+                resampling_strategy='test',
+                memory_limit=memory_lim,
+                disable_file_output=True,
+                logger=logger,
+                stats=stats,
```

Comment:
do we need to pass stats here or would it work without?

Reply:
It can work without it, but we are using it here because of the scenario configuration [here](https://github.com/automl/auto-sklearn/blob/25d680d4e9520a661aae48ea4c7295c663b64df3/scripts/run_auto-sklearn_for_metadata_generation.py#L132).

Is this scenario a must? If not, a dummy one is provided, which for the stats should be ok?